cropper/src/main/java/com/canhub/cropper/CropImage.kt

```diff
@@ -129,6 +129,7 @@ object CropImage {
      *
      * @param activity the activity to be used to start activity from
      */
+    @Deprecated("use the OpenChooser ActivityResultContract instead")
```

Comment:
I'm thinking better about the versioning of the library and this one will be a major release (congratz btw)

Instead of deprecating, maybe we could already delete. Any issue you can expect of forcing all users into contracts?

Reply:
If we keep it for now we get 2 benefits: 
- Users can only migrate partially which may be nice if they have a lot of usages
- IDE will show what to replace the deprecated methods with which is way nicer than having methods no longer existing